src/components/DocumentSelector/style.css

```diff
@@ -1,19 +1,24 @@
 @import (less) "../Theme/constants.css";
 
-.list {
-  list-style: none;
+.listContainer {
+  margin: 8*@unit @large-text-margin 0;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  padding: 0;
-  position: relative;
-  margin: 8*@unit @large-text-margin 0;
-  flex-shrink: 0;
+  justify-content: flex-start;
```

Comment:
Looks good. Do you think we need to extract this into a class and extend all the containers that need this type layout?

Reply:
Probably could if we think there's already a few containers that need it. Maybe as a separate PR?

Reply:
Slight change to the code based on See Wah's feedback.